FileMapper/pickerPrompt_script.html

```diff
@@ -0,0 +1,241 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   https://www.apache.org/licenses/LICENSE-2.0
+   
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<script>
+  // The Browser API key obtained from the Google API Console.
+  // Replace with your own Browser API key, or your own key.
+  var DEVELOPER_KEY = ENVIRONMENT.API_KEY;
+
+  // Google Picker
+  var DIALOG_DIMENSIONS = {width : 600, height : 425};
+  var pickerApiLoaded = false;
+  var localPathValue = document.getElementById("localpathinput").value;
+
+  // Regex expressions to validate absolute paths
+  var windowsPathRegExp = /^[\w]\:(\\|(\\[^<>\\/:"\|\?\*]+)+)\\?$/;
+  var unixPathRegExp = /^(\/[^<>\\/:"\|\?\*]+)*\/?$/;
+
+  /**
+   * Loads the Google Picker API.
+   */
+  function onApiLoad() {
+    gapi.load('picker', {'callback' : function() { pickerApiLoaded = true; }});
+  }
+
+  /**
+   * Gets the user's OAuth 2.0 access token from the server-side script so that
+   * it can be passed to Picker. This technique keeps Picker from needing to
+   * show its own authorization dialog, but is only possible if the OAuth scope
+   * that Picker needs is available in Apps Script. Otherwise, your Picker code
+   * will need to declare its own OAuth scopes.
+   */
+  function getOAuthToken() {
+    google.script.run
+        .withSuccessHandler(createPicker)
+        .withFailureHandler(showError)
+        .getOAuthToken();
+  }
+
+  /**
+   * Creates a Picker that can access the user's spreadsheets.
+   *
+   * @param {string} token An OAuth 2.0 access token that lets Picker access the
+   *     file type specified in the addView call.
+   */
+  function createPicker(token) {
```

Comment:
Is this duplicate of the method in other PRs? if yes we should merge them

Reply:
Actually the method name is same for all PRs but internally we have views for the File Picker changing. 
I can try making a single function in which we can send the view as an argument.

Reply:
we can have this method as a library and it can call / pass the displayMessage function which will be different for each of the different html pages? Any fix to this method need to be replicated in multiple places

Reply:
Yeah, I am working on this.

Reply:
I have created a separate client-side Picker Library which is shared across all the prompts.